www/client/config.js

```diff
@@ -1,6 +1,6 @@
 const API_URL = process.env.NODE_ENV === 'production'
   ? 'https://api.momentum-ui.com'
-  : 'http://127.0.0.1:3300';
+  : 'https://api.momentum-ui.com'; // http://127.0.0.1:3300
```

Comment:
@pauljeter would this be correct?

Reply:
I did not have local Word Press server. Will change back in the PR

Reply:
If you need to develop locally, you can add a .env file with:
API_URL=https://api.momentum-ui.com

Reply:
Got it! I've changed back the config file.